pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -417,6 +417,22 @@
   highlights_title: Highlights
   highlights_header: Hour of Code Highlights
   highlights_description: Some of our favorite moments from the Hour of Code campaign.
+  higlights_see_more: See more highlights
```

Comment:
I want to localize it since it's on the homepage. But OK that the rest of the highlights page isn't. I'll get to that later.